### ISSUE TITLE
UUID field names keep appearing when creating templates or elements

### ISSUE THREAD

**Issue by johardi** [metadatacenter/cedar-template-editor#848]
Steps to reproduce:
1. Create a new template and add a text field
2. Enter a name
3. Immediately save the template or add a new text field

The editor will show the field name as "Untitled" and a UUID will appear in the JSON document.

(This behavior will happen often if you run the steps above fast)-- Is there a way to increase the editor performance?

**Comment by martinjoconnor** [comment]
If you save the template does the UUID name go away?

**Comment by johardi** [comment]
No, it stays in the template and moreover, the UUID name will still persist even though you delete the field.

**Comment by willrett** [comment]
The way to improve the editor performance is to move to Angular 7 and a micro-service front end #951
